FirebaseMessaging/Sources/FIRMessagingExtensionHelper.m

```diff
@@ -62,6 +62,21 @@ - (void)populateNotificationContent:(UNMutableNotificationContent *)content
 }
 
 #if TARGET_OS_IOS || TARGET_OS_OSX || TARGET_OS_WATCH
+- (NSString *)fileExtensionForResponse:(NSURLResponse *)response {
+  NSString *const kDefaultExtension = @".png";
```

Comment:
Can you move both static string on top of this file following the other static parameters?